core-ui/src/components/BusolaExtensions/helpers.js

```diff
@@ -1,6 +1,15 @@
 import { prettifyKind } from 'shared/utils/helpers';
 import pluralize from 'pluralize';
-import { getLatestVersion } from '../../../components/Extensibility/migration';
+import { getLatestVersion } from 'components/Extensibility/migration';
+
+export const SECTIONS = [
```

Comment:
Why do we hide the version field? Now if user wants to update the version, he has to use yaml editor

Reply:
i thought the idea was to use the migration feature?